### PR TITLE
Upstream nov 19 2023

### PR DESCRIPTION
**Pull request by nevimer**

### CODE REVIEW

**Comment by StrangeWeirdKitten** [comment]
Commits look good on the Skyrat side of things. There hasn't been any balance shifting changes that came down since last month.

**Comment by YakumoChen** [comment]
except the quirk that gives you 6 free points

**Comment by Majkl-J** [comment]
Stuff this includes: Akula examine fix (Finally), BS Miner doesn't buzz loudly (Finally)
Oh and random rebalances to cargo pricing which may or may not be op

File: modular_skyrat/master_files/code/datums/quirks/negative.dm [hunk 1]
```diff
@@ -86,3 +86,15 @@
 	var/datum/quirk/blooddeficiency/synth/bd_synth = new
 	qdel(src)
 	return bd_synth.add_to_holder(new_holder, quirk_transfer)
+
+/datum/quirk/gifted
```
Comment (YakumoChen) [inline modular_skyrat/master_files/code/datums/quirks/negative.dm:90]: comment this out please, its garbage and shouldn't be on bubber

Comment (StrangeWeirdKitten) [reply modular_skyrat/master_files/code/datums/quirks/negative.dm:90]: Yes I'm aware of it.

Comment (StrangeWeirdKitten) [reply modular_skyrat/master_files/code/datums/quirks/negative.dm:90]: We're keeping this. I am going upstream later to make it a proper free points anyway

**Comment by StrangeWeirdKitten** [comment]
Today my hatred and itch to nuke ss13_construct from the codebase grows. But I've managed to suppress the urges of murder and I've finally beat the green check mark out of it and now I can finally forget about its existence. 

``/datum/unit_test/run_linters`` is not a unit test @TeshariEnjoer lol

**Comment by StrangeWeirdKitten** [comment]
This is good to merge now. @nevimer

**Pull request merged**